python/spacewalk/common/repo.py

```diff
@@ -84,7 +84,7 @@ def get(self, key: typing.Any) -> typing.Optional[typing.Any]:  # type: ignore
                 )
             return self[key]
 
-    def __init__(self, url: str, proxies: dict = None, gpg_verify: bool = True):
+    def __init__(self, url: str, proxies: dict = None, gpg_verify: bool = True, timeout: int = None):
```

Comment:
`None` is not an `int`, either the type hint or the default value is wrong.

Reply:
fixed